storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/ui/adapter/TweetsAdapter.java

```diff
@@ -34,29 +51,78 @@ public int getItemCount() {
     @Override
     public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
         return new ViewHolder(LayoutInflater.from(parent.getContext())
-                .inflate(R.layout.list_item_tweet, parent, false));
+                .inflate(R.layout.list_item_tweet, parent, false), storIOSQLite);
     }
 
     @SuppressLint("SetTextI18n")
     @Override
     public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
         final Tweet tweet = tweets.get(position);
 
+        holder.id = tweet.id();
         holder.authorTextView.setText("@" + tweet.author());
         holder.contentTextView.setText(tweet.content());
     }
 
     static class ViewHolder extends RecyclerView.ViewHolder {
 
+        private final StorIOSQLite storIOSQLite;
+
+        Long id;
```

Comment:
To the `Long id`?
It seems adding this one to `private final StorIOSQLite storIOSQLite` is excess, 'cause of `@NonNull StorIOSQLite storIOSQLite` in CTR, isn't?